onTruckPlugin/src/main/java/com/github/ontruck/controller/ManualController.java

```diff
@@ -12,7 +15,9 @@ public ManualController(IDriver driver) {
 		this.driver = driver;
 	}
 
-	// Process UDP Event
+	/**
+	 * Process driver command
```

Comment:
A bit too unspecific. Maybe something like 'Process driver command. It chooses how to send along the payload to the driver.'